packages/ui-kit/src/Toggler/Toggler.fixture.tsx

```diff
@@ -16,8 +17,14 @@ export default {
       foo: false,
       bar: true,
       baz: false,
+      payment: false,
     });
 
+    const classes = {
```

Comment:
why make a const for this if it's only used in one place? Can be defined inside the prop - that will be less confusing